### DISCUSSION TITLE
are borders working on Monterey ( intel )

### DISCUSSION THREAD

**Discussion by HosAkh** [asmvik/yabai#1074] [Q&A] [answered]
I don't want to install limelight. I have SIP disabled and posted config for my border below but I'm not getting any borders. Yabai is running and the config works as changing other things affects the windows managment. 

```
yabai -m config window_border_width          15
yabai -m config active_window_border_color   0xffC324
yabai -m config normal_window_border_color   0xa41313
yabai -m config insert_feedback_color        0x003cdb
```

**Comment by guidao** [comment, marked as answer]
https://github.com/koekeishiya/yabai/issues/1054

> Known issues:
> Window borders are broken; they cause random applications to freeze and also sometimes crashes the WindowServer. I have not > found a pattern to this behaviour. Most likely borders will be disabled for Monterey until a solution is found. Limelight needs to be > updated for Monterey; I'll look into that at some point, but no ETA.

**Reply by HosAkh** [reply]
:,(

**Reply by msr8** [reply]
I am using borders currently and the only time I have had some issues (minor, and they're known) is when finder is open. Can you please elaborate a bit on how they're broken? I dont want to find it out the hard way haha
